test/unit/fixtures/elm/ComparisonWithLiteral.json

```diff
@@ -0,0 +1,521 @@
+{
```

Comment:
Where did the CQL for these tests come from? It might be nice to have them incase we want/need to re-translate.

Reply:
I just made them myself. I still have them, would it be beneficial to commit them in the repository or just hang on to them?